<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManageChild</class>
 <widget class="QWidget" name="FileManageChild">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileManageChild</string>
  </property>
  <widget class="QListWidget" name="filelistWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="localFileBut">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>680</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>æ¥·ä½“</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);</string>
   </property>
   <property name="text">
    <string>æœ¬åœ°æ–‡ä»¶</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UdiskFileBut">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>680</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>æ¥·ä½“</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);</string>
   </property>
   <property name="text">
    <string>Uç›˜æ–‡ä»?/string>
   </property>
  </widget>
  <widget class="QTextEdit" name="filePrivewtextEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>551</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QScrollBar" name="filePreHorScrollBar">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>551</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="filePreVerScrollBar">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>10</y>
     <width>25</width>
     <height>301</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="loadSeleFileBut">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>æ¥·ä½“</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);</string>
   </property>
   <property name="text">
    <string>åŠ è½½é€‰ä¸­æ–‡ä»¶</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editSeleFileBut">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>480</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>æ¥·ä½“</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);</string>
   </property>
   <property name="text">
    <string>ç¼–è¾‘é€‰ä¸­æ–‡ä»¶</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delSeleFileBut">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>570</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>æ¥·ä½“</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);</string>
   </property>
   <property name="text">
    <string>åˆ é™¤é€‰ä¸­æ–‡ä»¶</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyFile2localBut">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>660</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>æ¥·ä½“</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);</string>
   </property>
   <property name="text">
    <string>å¤åˆ¶åˆ°æœ¬åœ?/string>
   </property>
  </widget>
  <widget class="QListWidget" name="ListWidget1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>351</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="filePrivewtextBro">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>571</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>ListWidget1</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>FileManageChild</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
